src/App.js

```diff
@@ -50,6 +50,18 @@ const INITIAL_DAO_STATE = {
   repos: [],
 }
 
+const SELECTOR_NETWORKS = [
+  ['main', 'Ethereum Mainnet', 'https://mainnet.aragon.org/'],
+  ['rinkeby', 'Ethereum Testnet (Rinkeby)', 'https://rinkeby.aragon.org/'],
+]
+if (network.type === 'ropsten') {
+  SELECTOR_NETWORKS.push([
+    'ropsten',
+    'Ethereum Testnet (Ropsten)',
+    'https://ropsten.aragon.org/',
```

Comment:
As discussed in https://github.com/aragonone/product/issues/146#issuecomment-508192951, only 'officially supported' networks should be in `.aragon.org`

```suggestion
    'https://aragon.ropsten.aragonpm.com/',
```